src/pages/programs/index.tsx

```diff
@@ -0,0 +1,94 @@
+    <ProgramsPageLargeParagraph
+      title="Our Programs"
+      description="ShelterTech's programs are designed to suppport the underserved community experiencing homelessness and housing insecurity through WiFi connectivity and acccess to online resources."
+    />
+    <ProgramsSubTitleBlock title="ShelterTech offers the following programs:" />
+    <SFServiceGuideBlock
+      title="SF Service Guide"
+      subtitle="an online directory of human services in San Francisco"
+      description="The SF Service Guide is an online directory of human services in San Francisco that launched in November 2019 and currently serves 1,500 unique monthly users. The directory's focus is on homelessness and housing services but also covers a variety of other services, from education and legal aid to senior services and re-entry programs. Our goal is to help anyone with access to smartphone, tablet, or computer to find services they need."
+      description2="SF Service Guide is supported by a grant from SF Mayor's Office of Housing and Community Development and verified for accuracy by our data partner, the Homeless Advocacy Project of the JDC."
+      image={{
+        url: classroom,
+        alt: "Classroom of volunteer members collaborating with one another.",
+      }}
+      ctaButtons={[
+        {
+          text: "Visit Site",
+          onClick: () => console.log(true),
+        },
+        {
+          text: "Donate",
+          onClick: () => console.log(true),
+        },
+      ]}
+    />
+    <ShelterConnectBlock
+      title="ShelterConnect"
+      subtitle="a program that installs free WiFi in overnight shelters, community centers, and supportive housing developments"
+      description="The ShelterConnect program works with partners to deliver a free, turn-key WiFi solution for shelters and transitional housing facilities. ShelterConnect is proud to provide over 4,000 people a year with daily internet access in over 20 local shelters and resource centers."
+      description2="We focus on identifying strategies to reduce the lifetime program cost for wiring a shelter. The three primary costs are:"
+      bullet1="Equipment and licensing"
+      bullet2="Installation labor"
+      bullet3="Ongoing ISP charges"
+      description3="In 2019 we established a partnership with the City of San Francisco Digital Equity team to deliver 1GB fiber/ISP capability to future ShelterConnect locations at no cost.  Working with partners, ShelterConnect has implemented free WiFi solutions at over 20 locations including the Star Hotel, Mission Hotel, MSC-South, and ten locations managed by Larkin Street Youth Services."
+      image={{
+        url: classroom,
+        alt: "Classroom of volunteer members collaborating with one another.",
+      }}
+      ctaButtons={[
+        {
+          text: "DONATE",
+          onClick: () => console.log(true),
+        },
+        {
+          text: "PARTNER WITH US",
+          onClick: () => console.log(true),
```

Comment:
Can we have this pop up the same partnership modal as on the Home Page? It's the one in the "Get Involved" section, under the "Partnerships" subsection.